app/src/main/java/org/sopt/melon/presentation/mixup/component/MixUpList.kt

```diff
@@ -0,0 +1,127 @@
+package org.sopt.melon.presentation.mixup.component
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.text.style.TextOverflow
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import org.sopt.melon.R
+import org.sopt.melon.core.designsystem.theme.MELONTheme
+import org.sopt.melon.presentation.mixup.data.MusicInfo
+
+@Composable
+fun MixUpList(
+    musicInfos: List<MusicInfo>,
+    modifier: Modifier = Modifier,
+) {
+    Column(
+        verticalArrangement = Arrangement.SpaceBetween,
+        modifier = modifier,
+    ) {
+        Text(
+            text = "믹스업 목록",
+            style = MELONTheme.typography.body.sb_16,
+            color = MELONTheme.colors.white,
+            maxLines = 1,
+            overflow = TextOverflow.Ellipsis,
+        )
+
+        LazyColumn(
+            contentPadding = PaddingValues(vertical = 16.dp),
+            verticalArrangement = Arrangement.spacedBy(16.dp),
+            modifier = Modifier,
+        ) {
+            items(8) { index ->
+                MixUpItem(
+                    musicInfo = musicInfos[index],
+                    isPlaying = musicInfos[index].isPlaying,
+                    isSelected = musicInfos[index].isSelected,
+                    onSelectClick = {},
+                    onMenuClick = {},
+                    modifier = Modifier.fillMaxWidth(),
+                )
```

Comment:
```suggestion
            items(musicInfos) { musicInfo ->
                MixUpItem(
                    musicInfo = musicInfo,
                    onSelectClick = {},
                    onMenuClick = {},
                    modifier = Modifier.fillMaxWidth(),
                )
```
p1: items 여기 하드코딩 되어 있는데 리스트 값을 전달해주세요 ~!! 그리구 musicInfos 자체에 프로퍼티 값이 isPlaying 과 isSelected 값이 들어있는거 같아서 이부분도 제거 하던지 아니면 분리해서 전달해주는 것이 좋을 거 같습니다 ~!!